src/components/header/signInHeader/signInHeader.js

```diff
@@ -243,42 +245,34 @@ class Header extends React.Component {
                             onClick={() => this.onChangeActiveNetwork(network.value)}
                           >
                             {network.label}
-                            <Input
-                              autoComplete="off"
-                              onChange={(value) => { this.changeAddress(value); }}
-                              name="customNetwork"
-                              value={this.state.address}
-                              placeholder={this.props.t('ie. 192.168.0.1')}
-                              size="s"
-                              className={`custom-network ${formStyles.input} ${this.state.validationError ? styles.errorInput : ''}`}
-                              onKeyDown={e => e.keyCode === keyCodes.enter
+                            <div className={styles.inputWrapper}>
+                              <Input
+                                autoComplete="off"
+                                onChange={(value) => { this.changeAddress(value); }}
+                                name="customNetwork"
+                                value={this.state.address}
+                                placeholder={this.props.t('ie. 192.168.0.1')}
+                                size="s"
+                                className={`custom-network ${styles.input} ${this.state.validationError ? styles.errorInput : ''}`}
+                                onKeyDown={e => e.keyCode === keyCodes.enter
                                 && this.onConnectToCustomNode(e)}
-                            />
-                            <div className={styles.icons}>
-                              <Spinner className={`${styles.spinner} ${this.state.isValidationLoading && this.state.address ? styles.show : styles.hide}`} />
-                              <Icon
-                                className={`${styles.status} ${!this.state.isValidationLoading && this.state.address && !this.state.isFirstTime
-                                  ? styles.show : styles.hide}`}
-                                name={!this.state.connected ? 'iconWarning' : 'okIcon'}
+                                isLoading={isValidationLoading && this.state.address}
+                                {...{}/* eslint-disable-next-line no-nested-ternary */}
+                                status={(!this.state.isValidationLoading
+                                    && this.state.address
+                                    && !this.state.isFirstTime)
+                                  ? (connected ? 'ok' : 'error')
+                                  : undefined
+                              }
+                                feedback={isActiveItem && validationError
```

Comment:
I know this is much simple but, IMO looks better the logic into a function, more than anything because of the ternary

Reply:
Solved by https://github.com/LiskHQ/lisk-hub/pull/2411/commits/b7425a155a5c38bad4aba2fe61b350fb2347dc1b